src/translator/service.h

```diff
@@ -161,6 +161,9 @@ class Service {
   /// Shortlist memory passed as bytes.
   AlignedMemory shortlistMemory_; // ORDER DEPENDENCY (translators_)
 
+  /// QE coefficients memory passed as bytes.
+  AlignedMemory qualityEstimator_; // ORDER DEPENDENCY (translators_)
```

Comment:
Jerin's notation is non-standard.  What he appears to mean by this is the class members need to be ordered in a particular way.  Does this apply to you?

Reply:
> Jerin's notation is non-standard

ummm... https://isocpp.org/wiki/faq/ctors#order-dependency-in-members. I just tag it with what the order dependency is on.

Reply:
I decided to remove it:  064cfe3